packages/resolve-scripts/configs/babelrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "presets": ["env", "react"],
+  "plugins": [
+    "add-module-exports",
```

Comment:
Is legacy CommonJS module requirement style still being actual in 2018 year? 
CommonJS legacy support had been introduced here:
https://github.com/reimagined/resolve/commit/c8de8a4efe6f87582a09d798220c114d9aa140d5
And automated by appropriate babel plugin here:
https://github.com/reimagined/resolve/commit/41276f95eeef9436c0dcb053d7c1a204575186a9